### ISSUE TITLE
Missing `content` property on page object

### ISSUE THREAD

**Issue by fortes** [sheebz/phantom-proxy#60]
The [PhantomJS colorwheel example](https://github.com/ariya/phantomjs/blob/master/examples/colorwheel.coffee) makes use of `page.content` in order to set document HTML.

Related: Looks like there are a few other properties missing from the Page object, per this document: https://github.com/ariya/phantomjs/wiki/API-Reference-WebPage#wiki-webpage-content

**Comment by skotchio** [comment]
Yes I have this problem too. But it seems that repos does not supported anymore :(

**Comment by dstevensio** [comment]
You can get this from page.get and page.set, I found today. 

``` js
page.open(uri, function () {
  var self = this;
  self.waitForSelector('body', function () {
    self.get('content', function (result) {
      var html = result.toString();
      // html contains string of HTML source
    });  
  }); 
});
```
